test/core/linter_test.py

```diff
@@ -406,8 +406,6 @@ def test__linter__empty_file():
         (
             False,
             [
-                ("L006", 3, 16),
-                ("L006", 3, 16),
                 ("L006", 3, 16),
```

Comment:
Even after deduplication, it looks like there are still 2 occurrences of:
```
("L006", 3, 16),
```

Any idea why?

Reply:
Ah - yeah I can see this is confusing. The rules we're checking is L006, which checks for whitespace *before* and *after* the operator. I've added a comment to explain this.

Reply:
Deduplication means that we only get 2 rather than 4.